extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -744,6 +765,44 @@ export class AmpA4A extends AMP.BaseElement {
   }
 
   /**
+   * Refreshes ad slot by fetching a new creative and rendering it. This leaves
+   * the current creative displayed until the next one is ready.
+   *
+   * @param {function()} refreshEndCallback When called, this function will
+   *   restart the refresh cycle.
+   */
+  refresh(refreshEndCallback) {
```

Comment:
Keith and I already went back and forth on this quite a bit. I think this function, at minimum, needs to be in this file, or else we'll be forced to expose various private fields that we'd prefer not to.